addon/ng2/commands/e2e.js

```diff
@@ -8,12 +8,8 @@ module.exports = Command.extend({
   name: 'e2e',
   description: 'Run e2e tests in existing project',
   works: 'insideProject',
-  run: function() {
-    var e2eTask = new E2ETask({
-      ui: this.ui,
-      analytics: this.analytics,
-      project: this.project
-    });
+  run: function () {
+    var e2eTask = new E2ETask({ ui: this.ui, analytics: this.analytics, project: this.project });
```

Comment:
is that a lint problem? if not, revert to the old code. it looks better with 1 line per entry.